Makefile

```diff
@@ -172,7 +180,7 @@ vagrantup: $(ALL) ## Run 'vagrant up' to (re-)provision a development environmen
 help: ## Show this help message
 	@echo "The following targets are available:"
 	@echo
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -Eh '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
grep + awk ? meh :/
Why not using a simple awk to do the job ?
```
@awk 'BEGIN { FS = ":.*?## " } /^[a-zA-Z_-]+:.*?## .*$$/{ printf "\033[36m%-30s\033[0m %s\n", $$1, $$2 }'
```

Reply:
We will soon have Perl oneliner, at this rate :p

Reply:
I would be the happiest man on earth :D

Reply:
Wrong PR for this :p